webapp/channels/src/components/post_view/reaction/__snapshots__/reaction.test.tsx.snap

```diff
@@ -1,50 +1,27 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`components/post_view/Reaction should apply read-only class if user does not have permission to add reaction 1`] = `
-<OverlayTrigger
-  defaultOverlayShown={false}
-  delayShow={500}
-  onEnter={[Function]}
-  overlay={
-    <Tooltip
-      id="post_id-smile-reaction"
-    >
-      <Memo(Connect(ReactionTooltip))
-        canAddReactions={false}
-        canRemoveReactions={true}
-        currentUserReacted={false}
-        emojiIcon={
-          <img
-            className="Reaction__emoji emoticon"
-            src="emoji_image_url"
-          />
-        }
-        emojiName="smile"
-        reactions={
-          Array [
-            Object {
-              "create_at": 0,
-              "emoji_name": ":smile:",
-              "post_id": "post_id",
-              "user_id": "user_id_2",
-            },
-            Object {
-              "create_at": 0,
-              "emoji_name": ":smile:",
-              "post_id": "post_id",
-              "user_id": "user_id_3",
-            },
-          ]
-        }
-      />
-    </Tooltip>
-  }
-  placement="top"
-  shouldUpdatePosition={true}
-  trigger={
+<Connect(ReactionTooltip)
```

Comment:
Yeah, it's intentional. This is one of those tests that just checks the snapshot of a shallowly rendered component without looking at the relevant fields here.

The only reason it's simpler is because the `WithTooltip` is now part of `ReactionTooltip` instead of before where it was a child of `Tooltip`